daemon/cmd/daemon_main.go

```diff
@@ -327,7 +327,7 @@ func initializeFlags() {
 	flags.String(option.EnablePolicy, option.DefaultEnforcement, "Enable policy enforcement")
 	option.BindEnv(Vp, option.EnablePolicy)
 
-	flags.Bool(option.EnableExternalIPs, defaults.EnableExternalIPs, fmt.Sprintf("Enable k8s service externalIPs feature (requires enabling %s)", option.EnableNodePort))
+	flags.Bool(option.EnableExternalIPs, false, fmt.Sprintf("Enable k8s service externalIPs feature (requires enabling %s)", option.EnableNodePort))
```

Comment:
do we need to mention upgrade note for these two flags ?

Reply:
Aye, I am planning to do in the follow up PR to update the docs (I want to get this PR merged before the freeze :grin: )